include/Core/Config.php

```diff
@@ -132,15 +132,33 @@ public static function set($family,$key,$value) {
 		$dbvalue = (is_array($value)?serialize($value):$value);
 		$dbvalue = (is_bool($dbvalue) ? intval($dbvalue) : $dbvalue);
 
-		$ret = q("INSERT INTO `config` ( `cat`, `k`, `v` ) VALUES ( '%s', '%s', '%s' )
-ON DUPLICATE KEY UPDATE `v` = '%s'",
+		// The "INSERT" command is very cost intense. It saves performance to do it this way.
```

Comment:
The "save_timestamp" is used by the "rendertime" addon which is used to detect code that needs to be optimized.

I do understand your concerns about the config stuff. Problem is that there are many systems out there that are running on small systems with bad I/O (like on shared hosters). My system as well has problems. I had times where each "set" in the old style lasted over two seconds.

What do you say to "REPLACE INTO"? It is not SQL standard but is said to be faster.